test/box-luatest/gh_8716_pagination_invalid_after_pos_bug_test.lua

```diff
@@ -0,0 +1,29 @@
+local server = require('luatest.server')
+local t = require('luatest')
+
+local g = t.group()
+
+g.before_all(function(cg)
+    cg.server = server:new()
+    cg.server:start()
+    cg.server:exec(function()
+        local s = box.schema.space.create('s')
+        s:create_index('i')
+    end)
+end)
+
+g.after_all(function(cg)
+    cg.server:drop()
+end)
+
+-- Checks that spaces with `_stream` and `_stream_space_cache` can be accessed
+-- from net.box streams.
+g.test_net_box_conn_with_disabled_schema_fetching = function(cg)
+    cg.server:exec(function()
+        local err_msg = "Iterator position is invalid"
+        t.assert_error_msg_content_equals(err_msg, function()
+            box.space.s:select(nil, {after = 777})
+            box.space.s:pairs(nil, {after = 777})
+        end)
```

Comment:
I guess you need to call this function twice since (for pairs and select). Currently `box.space.s:pairs(nil, {after = 777})` row is unreachable.

Reply:
Yeah, that's a typo, thanks!